SDL/MiniDoublet.cuh

```diff
@@ -727,6 +844,7 @@ namespace SDL
 
     struct createMDArrayRangesGPU
     {
+        ALPAKA_NO_HOST_ACC_WARNING
```

Comment:
I don't think these are necessary anymore after moving to a higher version of Alpaka, have to check.